apps/02RockPaperScissors.js

```diff
@@ -10,41 +10,40 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
+  // hand1 and hand are just variables
+
   if (hand1 === hand2) {
-  return "It's a tie!";
+    return "It's a tie!";
 }
 
   if (hand1 === 'rock') {
   if (hand2 === 'scissors') {
```

Comment:
You can shorthand your code using the `&&` oeprator:

example:

``if(taco === 'chicken' && drink === 'Mountain Dew') {
      return 'You had a chicken taco and Mountain Dew for lunch! Excellent decision making!'
}``